include/tag.hpp

```diff
@@ -57,7 +57,7 @@ struct alignas(hardware_constructive_interference_size) tag_t {
         return map.at(key);
```

Comment:
you could add `[[nodiscard]]' for this non-const-at as well (a bit nit-picky ...sorry).